linuxkit/aws_test.yml

```diff
@@ -0,0 +1,36 @@
+kernel:
+  image: linuxkit/kernel:4.9.51
+  cmdline: "console=tty0 console=ttyS0 console=ttyAMA0"
+init:
+  - linuxkit/init:7804129bd06218b72c298139a25698a748d253c6
+  - linuxkit/runc:a1b564248a0d0b118c11e61db9f84ecf41dd2d2a
+  - linuxkit/containerd:417f83f7b8dc1fa36acf90effe44f99c7397480a
+  - linuxkit/ca-certificates:e44b0a66df5a102c0e220f0066b0d904710dcb10
+onboot:
+  - name: sysctl
+    image: linuxkit/sysctl:154913b72c6f1f33eb408609fca9963628e8c051
+  - name: rngd1
+    image: linuxkit/rngd:558e86a36242bb74353bc9287b715ddb8567357e
+    command: ["/sbin/rngd", "-1"]
+  - name: dhcpcd
+    image: linuxkit/dhcpcd:f3f5413abb78fae9020e35bd4788fa93df4530b7
+    command: ["/sbin/dhcpcd", "--nobackground", "-f", "/dhcpcd.conf", "-1"]
+  - name: metadata
+    image: linuxkit/metadata:da3138079c168e0c5608d8f3853366c113ed91d2
+services:
+  - name: getty
+    image: linuxkit/getty:bf6872ce0a9f3ab519b3e502cc41ba3958bda2a6
+    env:
+     - INSECURE=true
+  - name: rngd
+    image: linuxkit/rngd:558e86a36242bb74353bc9287b715ddb8567357e
+  - name: sshd
+    image: linuxkit/sshd:d313eea3d9d7fbcbc927d06a6700325725db2a82
+files:
```

Comment:
Since we are installing aws metadata service, it will probably work if we set a keypair in the cloud formation and we let AWS inject it here.

Reply:
All instances are supposed to have the key injected by the CFN template, added this just in case to be sure it has the key. It can be removed.